hs/server.hs

```diff
@@ -16,11 +18,32 @@ import Database.CourseQueries (retrieveCourse, allCourses, queryGraphs, courseIn
 import Css.CssGen
 import Filesystem.Path.CurrentOS
 import System.Directory
+import System.Environment (lookupEnv)
+import System.IO (hSetBuffering, stdout, stderr, BufferMode(LineBuffering))
+import System.Log.Logger (logM, updateGlobalLogger, rootLoggerName, setLevel, Priority(INFO))
 import CourseographyFacebook
 import qualified Data.Text as T
 
+-- | log access requests using hslogger and a condensed log formatting
+--
+logMAccessShort :: FormatTime t => LogAccess t
```

Comment:
Note: see <https://hackage.haskell.org/package/happstack-server-7.4.3/docs/src/Happstack-Server-Internal-Types.html#logMAccess> for the default `logMAccess` function